protocols/ecmd/via_usart/ecmd_usart.c

```diff
@@ -52,7 +64,9 @@ ecmd_serial_usart_init(void) {
 
   RS485_TE_SETUP;             // configure RS485 transmit enable as output
   RS485_DISABLE_TX;           // disable RS485 transmitter
+#ifdef ECMD_USART_NEED_INIT
```

Comment:
Moved ifdef up and removed unneeded global vars initializiation.
Making meta.m4 init() depend on the same conditions than ECMD_USART_NEED_INIT would require xxx_USE_USART to be accessibly from meta - a major change touching several modules. Better count on gcc/linker optimizing the empty initializer away.